model/postgres/postgres_server.rb

```diff
@@ -47,13 +47,15 @@ def configure_hash
       lc_time: "'C.UTF-8'"
     }
 
-    if primary?
-      configs[:archive_mode] = "on"
-      configs[:archive_timeout] = "60"
-      configs[:archive_command] = "'/usr/bin/wal-g wal-push %p --config /etc/postgresql/wal-g.env'"
-    else
-      configs[:recovery_target_time] = "'#{resource.restore_target}'"
-      configs[:restore_command] = "'/usr/bin/wal-g wal-fetch %f %p --config /etc/postgresql/wal-g.env'"
+    if timeline.blob_storage_endpoint
```

Comment:
Do we have something to re-check that this is, in fact, a development environment? Seems to push the margin of safety towards not taking backups & not noticing.

Reply:
We have `Config.development?`, though where should we put the check? How about this;

```ruby
def blob_storage_endpoint
  @blob_storage_endpoint ||= blob_storage&.connection_strings&.first
  if @blob_storage_endpoint.nil? && Config.development?
    fail "BUG: Missing blob storage configuration"
end
```

Reply:
It could be this, it could also be something more indirect, such as paging if no backup is available. That is, noticing may be an adequate substitute for a branch.

Reply:
In particular, we don't notice BUGs right away, either.

Reply:
I understand the ask now. Actually adding "Missing Backups" page is in my "needs to be done before launch" list, though I prefer to do it in separate PR, otherwise scope of this PR (improving development experience) expands too much.

Is it OK to do paging separately?

Reply:
Merging this now. I'll do the paging separately.